tests/tasks/test_api_create_task.py

```diff
@@ -29,8 +29,7 @@ def test_create_task_api_no_description(self):
         auth_header = get_test_request_header(self.first_user.id)
         expected_response = messages.DESCRIPTION_FIELD_IS_MISSING
         actual_response = self.client.post(
-            f"/mentorship_relation/{self.mentorship_relation_w_second_user.id}"
-            "/task",
+            f"/mentorship_relation/{self.mentorship_relation_w_second_user.id}" "/task",
```

Comment:
Write it link `f"/mentorship_relation/{self.mentorship_relation_w_second_user.id}/task`